app/views/simple_captcha/_simple_captcha.haml

```diff
@@ -1,4 +1,2 @@
-.control-label
-  = simple_captcha_options[:image]
-.controls
-  = simple_captcha_options[:field]
\ No newline at end of file
+= simple_captcha_options[:image]
+= simple_captcha_options[:field]
```

Comment:
Yeah, that is why I suggested maybe separating the captcha out of the other form parts might make for better and easier to maintain design.